python/pip_install/pip_repository.bzl

```diff
@@ -475,6 +495,7 @@ def _pip_repository_impl(rctx):
         ]),
         "%%ANNOTATIONS%%": _format_dict(_repr_dict(annotations)),
         "%%CONFIG%%": _format_dict(_repr_dict(config)),
+        "%%ENTRYPOINT_REPO%%": "//{pkg}" if rctx.attr.incompatible_generate_aliases else "_{pkg}//",
```

Comment:
Something about this looks weird.

The variable is named "repo", but "//{pkg}" isn't a repo-name.

The "_{pkg}//" value: I dont't repos can start with underscore? I got an error when I tried to do that recently. Shouldn't it also have "@" in it?